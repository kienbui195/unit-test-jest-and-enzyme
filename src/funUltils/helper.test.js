import React from 'react'
import Link from './helper'
import renderer from 'react-test-renderer'

describe('<Link />', () => {
    it('render dung', () => {
        const root = renderer.create(<Link link={'https://fb.com'} nameLink={'Facebook'} />).toJSON()
        expect(root).toMatchSnapshot()
    })
})