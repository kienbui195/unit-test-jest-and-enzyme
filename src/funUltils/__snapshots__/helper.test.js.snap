// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link /> render dung 1`] = `
<a
  href="https://fb.com"
>
  Facebook
</a>
`;
